src/core/qgsproject.h

```diff
@@ -1214,6 +1222,9 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
     bool mTrustLayerMetadata = false;
 
     QgsCoordinateTransformContext mTransformContext;
+
+    // local temporary copies of downloaded UI form files
+    QList<QPointer<QTemporaryFile>> mUiFormLocalCopies;
```

Comment:
Does this need to be QPointers? As far as i can see they'll never get deleted anywhere except in the destructor of this class, right?

Reply:
Indeed